src/js/stores/validators/AppFormValidators.js

```diff
@@ -60,7 +60,7 @@ const AppFormValidators = {
       }),
 
   cpus: (value) => !Util.isStringAndEmpty(value) &&
-    !!value.toString().match(/^[0-9\.]+$/),
+    !!value.toString().match(/^[0-9\.]+$/) && value >= 0.01,
```

Comment:
We assume a number-type here, but maybe it's more save to apply an parseFloat(), so we are on the save side and cover all cases?
```javaScript
!!value.toString().match(/^[0-9\.]+$/) && parseFloat(value) >= 0.01
```
```